lib/lost-center.js

```diff
@@ -42,23 +65,27 @@ module.exports = function lostCenterDecl(css, settings) {
       lostCenterFlexbox = 'no-flex';
     }
 
-    decl.parent.nodes.forEach(function lostCenterPaddingFunction(declaration) {
-      if (declaration.prop === 'lost-center-padding') {
-        lostCenterPadding = declaration.value;
+    lostCenterPadding = lgLogic.parseLostProperty(decl.parent.nodes, 'lost-center-padding', lostCenterPadding);
+    lostColumnRounder = lgLogic.parseLostProperty(decl.parent.nodes, 'lost-column-rounder', lostColumnRounder);
```

Comment:
@self ...is the rounder required or could it have default? ...does `lost-center` need a rounder? ...I suppose if it's 100%? Think on this.